package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "acs-client",
+  "version": "1.0.0",
+  "description": "ACS Engine Client",
+  "scripts": {
+    "build": "tsc",
+    "watch": "tsc -w",
+    "lint": "tslint -c tslint.json -p tsconfig.json",
+    "start": "npm run build && electron ./dist/main.js",
+    "dev": "concurrently 'yarn start' 'yarn watch'",
+    "precommit": "lint-staged",
+    "prettier": "prettier --write 'src/**/*.{ts,tsx,css}'",
+    "lint-fix": "tslint -c ./tslint.ci.json --fix 'src/**/*.{ts,tsx}'"
+  },
+  "lint-staged": {
+    "*.{ts,tsx}": ["tslint -c ./tslint.ci.json --fix", "prettier --write", "git add"],
+    "*.{js,json,css,md}": ["prettier --write", "git add"]
+  },
+  "repository": "https://github.com/electron/electron-quick-start-typescript",
+  "keywords": ["Electron", "quick", "start", "tutorial", "demo", "typescript"],
+  "author": "GitHub",
+  "license": "CC0-1.0",
```

Comment:
Can you update those 4 rows, with the correct infos ?
Just want to make sure the first commit will be clean.